pages/tool_assembly/plant_genomics_assembly

```diff
@@ -0,0 +1,38 @@
+---
+title: Plant Genomics Assembly
+contributors: [Anne Françoise Adam Blondon, Cyril Pommier, Daniel Faria, Paulette Lieby, Sebastian Beier]
+summary: Tool assembly for managing plant genomic data
+tags: [plants]
+page_tag: plant_geno_assembly
+---
+
+## What is the plant genomics tool assembly?
+The plant genomics tool assembly is a toolkit for the management of plant genomics and genotyping data throughout its lifecycle, with a particular focus on ensuring traceability of the biological materials to enable interoperability with plant phenotyping data.
+
+## Who can use the plant genomics tool assembly?
+This tool assembly can be used by any researcher producing plant genomic or genotyping data interested in ensuring their data complies with the FAIR principles.
+
+## How can you access the plant genomics tool assembly?
+All the components of this tool assembly are publicly available, but most require registration. So anyone can access the tool assembly provided they register for each tool that requires it.
+
+
+## For what purpose can you use the plant genomics tool assembly?
+<!--- TODO: Add figure --->
+
+### Metadata collection and tracking
+Accurate description of the plant biological materials and samples is critical for interoperability, and should comply with the [MIAPPE](https://www.miappe.org/) standard.
+This information should be submitted to [BioSamples](https://www.ebi.ac.uk/biosamples/), with MIAPPE compliance validated using BioSamples' [plant-miappe.json](https://github.com/EBIBioSamples/biosamples-v4/blob/biohackathon_miappe_checklist/webapps/core/src/main/resources/schemas/certification/plant-miappe.json) template.
+Submission of sample descriptions to BioSamples can be done as early as the data collection stage, but at the latest, must acompany submission of the genomic data to the [European Nucleotide Archive](https://www.ebi.ac.uk/ena/browser/home) (ENA) or of genotyping data to the [European Variation Archive](https://www.ebi.ac.uk/eva/) (EVA).
+[e!DAL-PGP](https://edal-pgp.ipk-gatersleben.de/) can be used to manage and share experimental metadata, as well as data.
+
+### Data processing and analysis
+Reference genomes for genome assembly and annotation should be obtained from [ENSEMBL Plants](https://plants.ensembl.org/index.html), if available.
```

Comment:
Could this go to a reuse section?

Reply:
Do you mean the ENSEMBL Plants part? I'm not sure.
I tend to see the reuse section as ensuring the reuse of the data in my study, as opposed to reusing data from others in my study.